docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -35,6 +35,37 @@ function hs_layouts_plugin_filter_block__layout_builder_alter(&$definitions, &$e
   }
 }
 
+/**
+ * Implements hook_preprocess().
+ */
+function hs_layouts_preprocess(&$variables, $hook) {
```

Comment:
It might make sense to leave a code comment to explain what this is doing & why.

Reply:
@mndonx Thanks, fixed!